src/index.js

```diff
@@ -1,6 +1,17 @@
-const fibonacci = (n) => {
+const fibonacci = (n) =>{ 
+ 
+    let arr=[1];
+    let num1=1;
+    let num=0;
```

Comment:
use more descriptive variable names for function they will perform. For example `previousNumber`, `currentNumber`.